org/wildfly/undertow/http-invoker/host/capability.adoc

```diff
@@ -0,0 +1,38 @@
+NAME: org.wildfly.undertow.http-invoker.host
+
+DESCRIPTION: Http invoker host configuration
+
+REGISTERED BY:
+  
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: true
+
+SUPPORTS RUNTIME ONLY: true
+
+SERVICE PROVIDED:
+
+   CLASS: org.wildfly.extension.undertow.HttpInvokerHostService
+   MODULE: org.wildfly.extension.undertow
+
+HARD REQUIREMENTS:
+
+  NAME: org.wildfly.wildfly.http-invoker
+  DYNAMIC: false
+
+  NAME: org.wildfly.undertow.host
+  DYNAMIC: true
+
+OPTIONAL REQUIREMENTS:
+
+  NAME: org.wildfly.security.http-authentication-factory
+  DYNAMIC: true
+  DESCRIPTION: Needed when configured via http-authentication-factory attribute
+OPTIONAL REQUIREMENTS:
+
+  NAME: org.wildfly.modcluster
+  DYNAMIC: false
+  DESCRIPTION: Depends on mod cluster when mod cluster is installed in system
```

Comment:
no there is no config that enables this.
It just adds extra deps, when it is present.